cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -4345,6 +4367,26 @@ pub(crate) fn define(
         .operands_out(vec![a]),
     );
 
+    let x = &Operand::new("x", Float);
+    let a = &Operand::new("a", FloatTo);
+
+    ig.push(
+        Inst::new(
+            "fvpromote_low",
```

Comment:
For this one, I think it does make sense to keep the `_low` since it indicates which lanes get promoted, but I would drop the `v` since the `_low` already implies lanes.